admin/src/main/java/com/exadel/frs/scheduler/job/StatisticsJob.java

```diff
@@ -0,0 +1,92 @@
+package com.exadel.frs.scheduler.job;
+
+import com.exadel.frs.entity.Face;
+import com.exadel.frs.entity.Model;
+import com.exadel.frs.exception.ApperyServiceException;
+import com.exadel.frs.repository.FacesRepository;
+import com.exadel.frs.repository.InstallInfoRepository;
+import com.exadel.frs.repository.ModelRepository;
+import com.exadel.frs.system.feign.ApperyStatisticsClient;
+import com.exadel.frs.system.feign.StatisticsFacesEntity;
+import feign.FeignException;
+import lombok.NoArgsConstructor;
+import org.apache.commons.lang3.Range;
+import org.quartz.JobExecutionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.commons.lang3.Range.between;
+
+@NoArgsConstructor
+@Component
+public class StatisticsJob extends QuartzJobBean {
+
+    @Value("${app.feign.appery-io.api-key}")
+    private String statisticsApiKey;
+    private ApperyStatisticsClient apperyStatisticsClient;
+    private InstallInfoRepository installInfoRepository;
+    private FacesRepository facesRepository;
+    private ModelRepository modelRepository;
+
+    private List<Range> ranges = List.of(
+                Range.between(1, 10),
```

Comment:
Please make it 1-10, 11-50, 51-200, 201-500, 501-2000, 2001-10000, 10001-50000, 50001-200000, 200001-1000000, 1000001+

Reply:
fixed